modules/apps/layout/layout-locked-layouts-web/src/main/java/com/liferay/layout/locked/layouts/web/internal/portlet/action/UnlockLayoutsMVCActionCommand.java

```diff
@@ -29,8 +30,10 @@
  */
 @Component(
 	property = {
+		"javax.portlet.name=" + LockedItemsPortletKeys.LOCKED_ITEMS,
 		"javax.portlet.name=" + LockedLayoutsPortletKeys.LOCKED_LAYOUTS_PORTLET,
-		"mvc.command.name=/layout_locked_layouts/unlock_layouts"
+		"mvc.command.name=/layout_locked_layouts/unlock_layouts",
+		"mvc.command.name=/locked_items/unlock_layouts"
```

Comment:
Q: will we move this action to the new web module and make the constants class internal later?